.github/workflows/performance.yml

```diff
@@ -33,7 +33,7 @@ jobs:
   # Runs the performance test suite.
   performance:
     name: Performance tests ${{ matrix.memcached && '(with memcached)' || '' }}
-    uses: WordPress/wordpress-develop/.github/workflows/reusable-performance.yml@trunk
+    uses: desrosj/wordpress-develop/.github/workflows/reusable-performance.yml@fix/reusable-performance-workflow-target-sha
```

Comment:
Assume this is just for testing and will be updated prior to commit, but pointing it out just in case.

Reply:
Correct! Unfortunately, this is the only way to really test changes to reusable workflows because they reference specific repos and branches.